### PR TITLE
Add support for python3.7 and django2.2

### PR DESCRIPTION
**Pull request by jlward**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 language: python
-python: 3.5
+python: 3.7
+dist: xenial
```
Comment (jlward) [inline .travis.yml]: This is how we get travis to work with python3.7

File: mixpanel/tasks.py [hunk 2]
```diff
@@ -43,47 +42,49 @@ def run(self, event_name, properties=None, test=None, **kwargs):
         putting the events on a high-priority queue at Mixpanel for testing
         purposes.
         """
-        l = self.get_logger(**kwargs)
+        logger = self.get_logger(**kwargs)
```
Comment (jlward) [inline mixpanel/tasks.py]: Pretty much all the changes in this file were flake8

File: mixpanel/tests/test_tasks.py [hunk 3]
```diff
@@ -47,19 +47,40 @@ def set_mp_settings(self):
...
+                self._request_call_args.append(args)
+
+            @property
+            def request_call_args(self):
+                return self._request_call_args
+
+            def close(self, *args, **kwargs):
+                pass
         EventTracker._get_connection = lambda task: self.conn
-        self.response = Mock()
```
Comment (jlward) [inline mixpanel/tests/test_tasks.py]: I couldn't figure out how to get this working with mock. So I switched to something mock like. And updated everything to work with that.

To note, tests on master were failing for me. So it's not like I could jump back to debug.

File: mixpanel/tests/utils.py [hunk 4]
```diff
@@ -11,22 +11,22 @@
 def eager_tasks():
     app = app_or_default()
 
-    prev = app.conf.CELERY_ALWAYS_EAGER
-    app.conf.CELERY_ALWAYS_EAGER = True
+    prev = app.conf.task_always_eager
```
Comment (jlward) [inline mixpanel/tests/utils.py]: Celery changed how it stores it's settings.

**Comment by agriffis** [comment]
@jlward I didn't realize you were already a contributor... I was getting ready to ask if you wanted to be one :-)

Furthermore I thought I was a maintainer on PyPI, but it looks like it's you and @winhamwr.

This is all good news and a relief to me, since I haven't been active here for a while. Carry on! 😁

**Comment by jlward** [comment]
> This is all good news and a relief to me, since I haven't been active here for a while. Carry on! grin

No worries, I asked @winhamwr to give me access yesterday so I could make sure this lib was ready for python3.7 and django2.2. It probably was before this PR, but the tests needed to be updated due to some weird changes in kombu. But now we have passing tests and a new version on PyPi.

**Pull request merged**